helper/duration/duration.go

```diff
@@ -0,0 +1,28 @@
+package duration
+
+import (
+	"strconv"
+	"strings"
+	"time"
+)
+
+func ParseDurationSecond(inp string) (int, error) {
+	var result int
+	// Look for a suffix otherwise its a plain second value
+	if strings.HasSuffix(inp, "s") || strings.HasSuffix(inp, "m") || strings.HasSuffix(inp, "h") {
```

Comment:
@sergeyklay As the type indicates, this goes down only to a granularity of one second, so only `s`/`m`/`h` are handled.